app/models/match_decisions/eight/eight_record_voucher_date.rb

```diff
@@ -90,12 +88,12 @@ def notify_contact_of_action_taken_on_behalf_of contact: # rubocop:disable Lint/
 
     def accessible_by? contact
       contact.user_can_act_on_behalf_of_match_contacts? ||
-      contact.in?(match.housing_subsidy_admin_contacts)
+        contact.in?(match.send(contact_actor_type))
     end
 
-    def to_param
-      :eight_record_voucher_date_housing_subsidy_admin
-    end
+    # def to_param
+    #   :eight_record_voucher_date_housing_subsidy_admin
+    # end
 
```

Comment:
This looks like it was an abandoned direction for the code?
```suggestion
```

Reply:
Every class had these, we use them, whenever we're asking it to render, but 99% of them were correct (or should have been correct) by convention.  Guess I didn't fully clean this up.